.github/workflows/send-calypso-app-build-trigger.sh

```diff
@@ -1,29 +1,34 @@
 #!/bin/bash
 set -Eeuo pipefail
 
-trigger_payload=`cat $GITHUB_EVENT_PATH`
-
-workflow_data="{
-	\"action\": \"$GITHUB_ACTION\",
-	\"actor\": \"$GITHUB_ACTOR\",
-	\"run_id\": \"$GITHUB_RUN_ID\",
-	\"run_num\": \"$GITHUB_RUN_NUMBER\",
-	\"repo\": \"$GITHUB_REPOSITORY\",
-	\"trigger_payload\": $trigger_payload
-}"
-
 # cd here so that the parent directories are not included in the zip file.
 cd apps/full-site-editing/full-site-editing-plugin
 
 # Create a zip of the FSE plugin. Should include built files at this point.
 build_archive=plugin-archive.zip
-zip -r $build_archive ./*
+zip --quiet --recurse-paths $build_archive ./*
+
+
+# Use node to process data into JSON file
+node -e '
+const fs = require("fs");
+const trigger_payload = JSON.parse( fs.readFileSync( process.env.GITHUB_EVENT_PATH, "utf8" ) );
+const output = JSON.stringify( {
+	action: process.env.GITHUB_ACTION,
```

Comment:
While we're at it, can we do a quick check to make sure these env variables are defined on this side of things?